packages/commerce-sdk-react/src/auth/storage/cookie.ts

```diff
@@ -31,13 +31,45 @@ export class CookieStorage extends BaseStorage {
     }
     get(key: string) {
         const suffixedKey = this.getSuffixedKey(key)
-        return Cookies.get(suffixedKey) || ''
+        let value = Cookies.get(suffixedKey) || ''
+
+        // TODO: How can we automate a test for this?
+        if (value) {
+            // Some values, like the access token, may be split
+            // across multiple keys to fit under ECOM cookie size
+            // thresholds. We check for and append additional chunks here.
+            let chunk = 2
+            let additionalPart = Cookies.get(`${suffixedKey}_${chunk}`)
+            while (additionalPart) {
+                value = value.concat(additionalPart)
+                chunk++
+                additionalPart = Cookies.get(`${suffixedKey}_${chunk}`) || ''
+            }
+        }
+
+        return value
     }
     delete(key: string, options?: Cookies.CookieAttributes) {
         const suffixedKey = this.getSuffixedKey(key)
+
         Cookies.remove(suffixedKey, {
             ...getDefaultCookieAttributes(),
             ...options
         })
+
+        // Some values, like the access token, may be split
+        // across multiple keys to fit under ECOM cookie size
+        // thresholds. We check for and delete additional chunks here.
+        // TODO: How can we automate a test for this?
+        let chunk = 2
+        let additionalPart = Cookies.get(`${suffixedKey}_${chunk}`)
+        while (additionalPart) {
+            Cookies.remove(`${suffixedKey}_${chunk}`, {
+                ...getDefaultCookieAttributes(),
+                ...options
+            })
+            chunk++
+            additionalPart = Cookies.get(`${suffixedKey}_${chunk}`) || ''
+        }
```

Comment:
I noticed that the "set" function doesn't handle chunking. I feel like we should probably implement that for symmetry's sake. Lets talk if you have questions about that.

Reply:
EDIT: .. is that an SRFA only limitation.. or a cookie limitations.. I'm guessing the former?

Reply:
The `cc-at` cookie where we handle chunking is currently a one-way street from SFRA -> PWA. Nothing in plugin_slas reads the `cc-at` cookie so there is no requirement for us to set this cookie in the PWA. PWA -> SFRA session sharing happens via the `dwsid` instead.

More generically, the PWA does not set any cookies that exceed ecom thresholds (and I don't think we are planning to anytime soon). 
The cookie chunking exists only to address a current ecom limitation. For projects not interacting with ecom (ie. PWA only projects), the chunking could be a bit weird to have around.